### PR TITLE
LPD-13569   Create Stripe Payment Integration Client Extension - part 1

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/LPD-13569

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

File: workspaces/liferay-stripe-workspace/client-extensions/liferay-stripe-payment-integration/src/main/resources/application-default.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
+# LXC
+#
+
+com.liferay.lxc.dxp.domains=localhost:8080
+com.liferay.lxc.dxp.mainDomain=localhost:8080
+com.liferay.lxc.dxp.server.protocol=http
```
Comment (luca-pellizzon) [inline workspaces/liferay-stripe-workspace/client-extensions/liferay-stripe-payment-integration/src/main/resources/application-default.properties:7]: remove them

File: workspaces/liferay-stripe-workspace/client-extensions/liferay-stripe-payment-integration/client-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+assemble:
+    - fromTask: bootJar
+liferay-stripe-payment-integration:
+    key: liferay-stripe-payment-integration
+    name: Liferay Stripe Payment Integration
```
[the remark this answers is not in the document]
Comment (CrescenzoRegaLF) [reply workspaces/liferay-stripe-workspace/client-extensions/liferay-stripe-payment-integration/client-extension.yaml:5]: If you are referring to the initial part of the name (Liferay) I looked at all the workspaces where there are client extensions and they all have that naming convention

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**